### PR TITLE
Add swagger and documentation for V1 REST API

### PR DESCRIPTION
**Pull request by xscript**

### CODE REVIEW

File: docs/swagger/v1.json [hunk 1]
```diff
@@ -0,0 +1,217 @@
+{
```
Comment (davidfowl) [inline docs/swagger/v1.json]: No yaml?

Comment (xscript) [reply docs/swagger/v1.json]: No.
We are following Azure Management REST APIs to use `json`. Is `yaml` preferred over `json`?

Comment (davidfowl) [reply docs/swagger/v1.json]: I actually have no idea, I thought open API went to yaml instead of json.

Comment (xscript) [reply docs/swagger/v1.json]: Both of them are okay according to [here](https://swagger.io/specification/#format).

File: docs/rest-api.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+`1.0` | `PUT` | `https://<instance-name>.service.signalr.net/api/v1/hubs/<hub-name>/groups/<group-name>/users/<user-id>`
+
+<a name="remove-user-from-group"></a>
+### Remove a user from a group
+
+API Version | HTTP Method | Request URL
+---|---|---
+`1.0` | `DELETE` | `https://<instance-name>.service.signalr.net/api/v1/hubs/<hub-name>/groups/<group-name>/users/<user-id>`
+
+## Using REST API
```
[the remark this answers is not in the document]
Comment (xscript) [reply docs/rest-api.md]: Already have a link to our sample project at the architecture section. Also sample code is not trivial enough to put here. I can add a section to point to the other sample project we have.

File: docs/rest-api.md [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+
+<a name="user-api"></a>
+### User-related REST API
+
+In order to call user-related REST API, each of your clients should identify themselves to Azure SignalR Service.
```
Comment (chenkennt) [inline docs/rest-api.md]: Add more details about how this can be done (through negotiate). Add an example about negotiate response.

Comment (xscript) [reply docs/rest-api.md]: Negotiate will be explained in details in my next PR. Will reference it then,

File: docs/rest-api.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+    - [Authentication](#authentication)
+        - [Signing Algorithm and Signature](#signing)
+        - [Claims](#claims)
+    - [User-related REST API](#user-api)
+    - [Sample](#sample)
+
+On top of classical client-server pattern, Azure SignalR Service provides a set of REST APIs, so that you can easily integrate real-time functionality into your server-less architecture.
+
+<a name="serverless"></a>
+## Typical Server-less Architecture with Azure Functions
```
Comment (chenkennt) [inline docs/rest-api.md]: Add a note that 1.1.0-preview client cannot work in serverless scenario

Comment (xscript) [reply docs/rest-api.md]: Sure.

**Comment by xscript** [comment]
:alarm_clock: :alarm_clock: :alarm_clock:
Ping...

**Pull request merged**